docs/guides/real-time/chat/javascript.md

```diff
@@ -254,72 +304,85 @@ This guide covers authentication, item creation, and subscription using WebSocke
 		</form>
 
 		<script>
-			const url = 'wss://your-directus-url/websocket';
-			let connection;
-
-			document.querySelector('#login').addEventListener('submit', function (event) {
-				event.preventDefault();
-				const email = event.target.elements.email.value;
-				const password = event.target.elements.password.value;
-				connection = new WebSocket(url);
-				connection.addEventListener('open', function () {
-					connection.send(
-						JSON.stringify({
-							type: 'auth',
-							email,
-							password,
-						})
-					);
-				});
-				connection.addEventListener('message', function (message) {
-					receiveMessage(message);
-				});
-			});
-
-			document.querySelector('#new').addEventListener('submit', function (event) {
-				event.preventDefault();
-				const text = event.target.elements.text.value;
-				connection.send(
-					JSON.stringify({
-						type: 'items',
-						collection: 'messages',
-						action: 'create',
-						data: { text },
-					})
-				);
-				document.querySelector('#text').value = '';
-			});
-
-			function receiveMessage(message) {
-				const data = JSON.parse(message.data);
-				if (data.type == 'auth' && data.status == 'ok') {
-					connection.send(
-						JSON.stringify({
-							type: 'subscribe',
-							collection: 'messages',
-							query: {
-								fields: ['*', 'user_created.first_name'],
-								sort: 'date_created',
-							},
-						})
-					);
-				}
-				if (data.type == 'subscription' && data.event == 'init') {
-					for (const message of data.data) {
-						addMessageToList(message);
-					}
-				}
-				if (data.type == 'subscription' && data.event == 'create') {
-					addMessageToList(data.data[0]);
-				}
-			}
-
-			function addMessageToList(message) {
-				const li = document.createElement('li');
-				li.setAttribute('id', message.id);
-				li.textContent = `${message.user_created.first_name}: ${message.text}`;
-				document.querySelector('ol').appendChild(li);
-			}
+		  import {
+        createDirectus,
+        authentication,
+        realtime,
+      } from 'https://www.unpkg.com/@directus/sdk/dist/index.js';
```

Comment:
Indentation has gone a bit wonky here in the full example.

Reply:
Or, rather, is inconsistent. I prefer the one-liner for this.